Formula/pngcheck.rb

```diff
@@ -1,14 +1,9 @@
 class Pngcheck < Formula
   desc "Print info and check PNG, JNG, and MNG files"
   homepage "http://www.libpng.org/pub/png/apps/pngcheck.html"
-  url "https://downloads.sourceforge.net/project/png-mng/pngcheck/2.3.0/pngcheck-2.3.0.tar.gz"
-  sha256 "77f0a039ac64df55fbd06af6f872fdbad4f639d009bbb5cd5cbe4db25690f35f"
-  revision 1
-
-  livecheck do
-    url :stable
-    regex(%r{url=.*?/pngcheck[._-]v?(\d+(?:\.\d+)+)\.t}i)
-  end
```

Comment:
We could check for `http://www.libpng.org/pub/png/src/pngcheck-<VERSION_REGEX>.CHANGELOG` on the page

Reply:
I can't create a suggestion for deleted lines but the `livecheck` block should be updated as follows:

```ruby
livecheck do
  url :homepage
  regex(/href=.*?pngcheck[._-]v?(\d+(?:\.\d+)+)\.t/i)
end
```